solved

Krasul1a opened the pull request.

Krasul1a committed:
solved

vsmutok commented on app/main.py:
```diff
@@ -2,5 +2,16 @@
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cah = {}
```
`cah` not clear variable name

vsmutok commented on app/main.py:
```diff
@@ -2,5 +2,16 @@
+    def warp(*args) -> int:
+        if args in cah:
+            print("Getting from cache")
+            return cah[args]
```
* use `functools.wraps`
* what  if the function takes the same `args ` but different `kwargs` 
* you must use `args ` with `kwargs`  as the key of `cah `

Krasul1a committed:
1) rename cah to storage_cache,
2) added hash_key so that it can be used at the same time *args, **kwargs,
3) use functools.wraps
